haystack/json-schemas/haystack-pipeline-main.schema.json

```diff
@@ -3043,24 +3054,15 @@
               "title": "Use System Proxy",
               "type": "boolean"
             },
-            "embed_meta_fields": {
```

Comment:
I wonder why this is shown as a change in this PR? For `api_key` I understand that it just changed its position in the file due to the newly introduced `json.dump(sort_key=True)`. However, for `embed_meta_fields` I don't understand why there is this change here. The line is still part of the current main branch: https://github.com/deepset-ai/haystack/blob/50f34372e15fc38a512d0e268714c0feb9d540be/haystack/json-schemas/haystack-pipeline-main.schema.json#L3046 so I guess we don't want this change to be merged, do we?

Reply:
Sounds good, thanks for the review!